app/assets/javascripts/discourse/app/controllers/user.js

```diff
@@ -198,6 +198,17 @@ export default Controller.extend(CanCheckEmails, {
     this.toggleProperty("displayUserNav");
   },
 
+  get displayTopLevelAdminButton() {
+    if (!this.currentUser.staff) {
```

Comment:
Is `this.currentUser` guaranteed to be present here?

Reply:
🤦 fixed, thanks!